README.md

```diff
@@ -1,56 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Project Name: Hello World
 
-> Description the project.
+> Description the project: This is a project to setup the environment using linters to control syntax error, page structure etc.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+- Major languages: HTML/CSS
+- Frameworks: None
+- Technologies used: None
 
 
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Ernest Anyewe Adonu**
 
-👤 **Author2**
+- GitHub: [@principlebrothers]
+- LinkedIn: [Ernest Anyewe Adonu]
```

Comment:
- [ ] Please put your actual social links for GitHub, LinkedIn, and Twitter to make your README file more professional